target/linux/ramips/image/mt7621.mk

```diff
@@ -2273,6 +2273,16 @@ define Device/rostelecom_rt-sf-1
 endef
 TARGET_DEVICES += rostelecom_rt-sf-1
 
+define Device/ruijie_rg-ew1200g-pro-v1.1
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 15808k
+  DEVICE_VENDOR := Ruijie
+  DEVICE_MODEL := RG-EW1200G PRO v1.1
```

Comment:
`v1.1` should probably be moved to `DEVICE_VARIANT`.

Reply:
Done